vit-servicing-station-tests/src/tests/graphql/proposals.rs

```diff
@@ -9,6 +9,7 @@ use askama::Template;
 use assert_fs::TempDir;
 
 #[test]
+#[ignore]
```

Comment:
So, no luck getting it pass?

Reply:
I didn't try with these latest changes. Anyway, it was ignored because of the proposition of getting it removed, which is probably what I am gonna do after this one is merged. But I can get it a try for the completeness of the lib state.

Reply:
It's OK to delete the polymorphic fields from GraphQL and/or the tests, we can re-add them later and with proper fragment support in the schema if needed.

Reply:
Yeah, the issue is not doing or not the implementation. But to actually test it. Current serde implementation with `flatten+alias` does not work. If we are gonna drop the GraphQL support we can completely skip this.